src/viewport_manager.ts

```diff
@@ -87,6 +93,27 @@ export class ViewportManager implements Disposable {
                         view.line = curline;
                         view.col = curcol;
                     }
```

Comment:
off-topic, line 83 (89) looks weird, is it intentional?

                        for (const [grid, , topline, botline, curline, curcol] of args) {

Reply:
Yes, currently the second item(win) is not needed